### PR TITLE
Roman to Arabic

### PR DESCRIPTION
**Pull request by davidtran**

### CODE REVIEW

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 1]
```diff
@@ -1,6 +1,62 @@
...
 
-  # code code code
-  # what? why?
+        @subtractPrefixes = [
+            'I', 'X', 'C'
+        ]
 
-end
\ No newline at end of file
+        @romanArray = [
```
Comment (joneslee85) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: what do you think about using `@romanMap.keys` instead?

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 2]
```diff
@@ -1,6 +1,62 @@
 class RomanNumeral
+    @result
+    def initialize input
+        @result = 0
+        @input = input
+        @romanMap = {
```
Comment (joneslee85) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: FYI, majority of Ruby developers uses snake case

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 3]
```diff
@@ -1,6 +1,62 @@
...
+                        flag = true
+                    end
+                end
+            end
+
+            @result += @romanMap[char] if !flag
+
+            i += 1
+        end
+        @result
```
Comment (joneslee85) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: looks good :+1:

File: roman_to_arabic_numeral/roman_to_arabic.rb (hunk 2 above)
Comment (rauxalach) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: bring @romanMap to CONST

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 4]
```diff
@@ -1,6 +1,62 @@
 class RomanNumeral
+    @result
```
Comment (rauxalach) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: why you need to make @result to be accessible outside the class ?

If you do so, should it be ```attr_accessor :result``` ? Ruby class syntax

File: roman_to_arabic_numeral/roman_to_arabic.rb [hunk 5]
```diff
@@ -1,6 +1,62 @@
...
+            'C',
+            'D',
+            'M'
+        ]
+
+    end
+
+
+    def to_arabic
+        @result = 0
```
Comment (rauxalach) [inline roman_to_arabic_numeral/roman_to_arabic.rb]: haven't you set @result = 0 at initialize stage ?

**Pull request closed without merging**